razzle.config.js

```diff
@@ -186,4 +114,23 @@ module.exports = Object.assign({}, volto_config, {
 
     return base_config;
   },
+  plugins: [
```

Comment:
Parlando con Giulia mi è venuto in mente che forse qui c'è un bug.
Scritto così, la chiave ``plugins`` di ``volto_config`` viene sovrascritta del tutto, mentre vogliamo solo aggiungerne un pezzo. Potenzialmente è sufficiente spreaddare ``volto_config`` all'inizio di questo array. Che ne pensi @nzambello ?